### PR TITLE
Hexagon Aerial Imagery blocks

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Pull Request
============

> Tracked in JIRA by [UP-3388](https://geoinformationstore.atlassian.net/browse/UP-3388)

### Scope of the PR

Add docs for Hexagon Aerial Imagery 15 and 30cm blocks.

### Checklist

- [x] Add max area for query
- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Add relevant changes to [changelog](./source/changelog.rst)
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/data/hexagon-aerial-15cm.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
+.. meta::
+   :description: UP42 data blocks: Hexagon 15cm Aerial Imagery AOI clipped block description
+   :keywords: Hexagon, Aerial, AOI clipped, block description
+
+.. _hexagon-aerial-15cm-block:
+
+Hexagon 15cm Aerial Imagery AOI clipped
+=======================================
+`Link <https://marketplace.up42.com/block/045019bb-06fc-4fa1-b703-318725b4d8af>`_ to block details page
```
Comment (rodrigoalmeida94) [inline source/up42-blocks/data/hexagon-aerial-15cm.rst:9]: @perusio @nb-42 not sure what this link should be?

File: source/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,288 @@
+.. meta::
```
Comment (perusio) [inline source/changelog.rst:1]: Please do not commit the changelog. This is quite an old PR, please be mindful of re-introducing cruft. Thank you.

File: source/up42-blocks/data/hexagon-aerial-15cm.rst (hunk 1 above)
Comment (perusio) [reply source/up42-blocks/data/hexagon-aerial-15cm.rst:9]: Is up to @nb-42 to create the entry in the marketplace, then we can have the proper link.

File: source/up42-blocks/data/hexagon-aerial-30cm.rst [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+* ``bbox``: The bounding box to use as an AOI. Will return all scenes that intersect with this box. Use only ``bbox``
+  **or** ``intersects`` **or** ``contains``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use only ``bbox``
+  **or** ``intersects`` **or** ``contains``.
+* ``zoom_level``: An integer defining the :term:`Web Mercator` zoom level of this request, defaults to 19.
+
+.. warning::
+
+  The maximum AOI size supported by this block is **100 sqkm**.
+
```
Comment (perusio) [inline source/up42-blocks/data/hexagon-aerial-30cm.rst:29]: Can we use superscripts here: 
```
100 km :superscript:`2`
```

File: source/up42-blocks/data/hexagon-aerial-15cm.rst [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+* ``bbox``: The bounding box to use as an AOI. Will return all scenes that intersect with this box. Use only ``bbox``
+  **or** ``intersects`` **or** ``contains``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use only ``bbox``
+  **or** ``intersects`` **or** ``contains``.
+* ``zoom_level``: An integer defining the :term:`Web Mercator` zoom level of this request, defaults to 20.
+
+.. warning::
+
+  The maximum AOI size supported by this block is **25 sqkm**.
+
```
Comment (perusio) [inline source/up42-blocks/data/hexagon-aerial-15cm.rst:29]: Ditto from below:
```
25 km :superscript: `2`
```

**Review by perusio — CHANGES_REQUESTED** [review]
Please see comments below. Thank you.

File: source/changelog.rst (hunk 2 above)
Comment (rodrigoalmeida94) [reply source/changelog.rst:1]: Sorry, must have slipped in the rebase! Thanks

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**